### PR TITLE
Matt Toensing 😝

### PR DESCRIPTION
**Pull request by matttoensing**
I struggled with the recommend personnel method. I also had some questions about which person was supposed to be in the correct hash value in iteration 3. I left comments. Thank you instructors!!

### CODE REVIEW

File: lib/flotilla.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  def recommend_personnel(ship) # I think I messed this up!
+    recommend = []
+    @personnel.each do |person|
+      ship.requirements.each do |requirement|            # X is bad but couldn't think of name 
+        recommend << person if person.specialties.any?{|x|  requirement.keys.include?(x)} && person.experience > requirement.values.sum
+      end
+    end
+    recommend = recommend.uniq
+    recommend
+  end
```
Comment (mikedao) [inline lib/flotilla.rb:28]: This is right on, you got it. I would have probably pulled out the inner iteration to another method to simplify it up

File: lib/flotilla.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    @ships.each do |ship|
+      people_in_ship[ship] = recommend_personnel(ship)
+    end
+    people_in_ship
+  end
+
+  def ready_ships(fuel)
+    ready = []
+    personnel_by_ship.each do |ship, personnel|
+      ready << ship if ship.fuel > fuel && personnel.length >= ship.requirements.length
```
Comment (mikedao) [inline lib/flotilla.rb:41]: I like how you checked to make sure all requirements were met here.

**Review by mikedao — COMMENTED** [review]
Overall just super stellar work here.

Score: pass

**Comment by matttoensing** [comment]
Thank you Mike!!

**Pull request closed without merging**